rules/template/ConflateRules.js

```diff
@@ -99,10 +99,10 @@ exports['isMatchCandidate'] = isMatchCandidate;
  * review scores. Values that are omitted are assumed to be zero. If the result
  * is a review then an explanation must be given in the 'explain' key. The
  * following are all valid results:
- *
- * - `return {'match': 1};`
- * - `return {'match': 0.5, 'miss': 0.5, 'review': 0};`
- * - `return {'review': 1, 'explain': "This doesn't look right."};`
+ * 
+ * - return {"match": 1};
+ * - return {"match": 0.5, "miss": 0.5, "review": 0};
+ * - return {"review": 1, "explain": "This doesn't look right."};
```

Comment:
This file is parsed and put into the PDF via ASCIIDOC and this made it more readable.